modules/vba_documents/app/workers/vba_documents/upload_processor.rb

```diff
@@ -90,6 +91,10 @@ def download_and_process
     end
     # rubocop:enable Metrics/MethodLength
 
+    def base64_encoded?(tempfile)
+      VBADocuments::MultipartParser.base64_encoded?(tempfile.path)
```

Comment:
I considered moving the `base64_encoded?` functionality out of `VBADocuments::MultipartParser` and into `VBADocuments::UploadProcessor`, but `VBADocuments::MultipartParser.parse` is called by several other classes, and so I would have had to duplicate the functionality if I'd done so. The implementation I chose here seemed like the lesser of two evils.